js/everything.js

```diff
@@ -509,11 +594,52 @@ $(function() {
 
     errorMessage.init();
 
+    videoMetaData = window.sessionStorage.getItem("videoMetaData");
+
+    if (videoMetaData !== null ) {
+        videoMetaData = JSON.parse(videoMetaData);
+        autoplayState = videoMetaData["autoplayState"];
+    }
+    if (autoplayState !== null)
+    {
+        if (autoplayState === "true")
```

Comment:
This if/else should be refactored into a function called `updateAutoplayToggle(state)`, where the `state` parameter is a boolean